docs/tutorials/seurat-de/seurat-de.md

```diff
@@ -741,11 +732,9 @@ after.harmony | after.seuratCCA
 !!! question
     What do you think of the integration results now?
 
-    **Challenge:** look at the pc1/2 plots for each integration method -\> explain different pca slots here
+    **Hint:** Also look at the pc1 and pc2 plots for each integration method
```

Comment:
Minor suggestion but capitalise PC1 and PC2 and add punctuation (period `.`) at the end of sentences.